### PR TITLE
add local_tool_data_table_conf.xml

### PR DESCRIPTION
**Pull request by mthang**
Add custom database for dev machine

### CODE REVIEW

**Comment by mthang** [comment]
Add local_tool_data_table_conf.xml to files/galaxy/config/ repo
Add local_tool_data_table_conf.xml entry to dev.usegalaxy.org.au.xml

**Comment by mthang** [comment]
updated dev.usegalaxy.org.au.yml

**Comment by mthang** [comment]
add tool_data_table_config_path variable to dev.usegalaxy.org.au.yml

File: host_vars/dev.usegalaxy.org.au.yml [hunk 1]
```diff
@@ -102,6 +102,8 @@ host_galaxy_config_files:
     dest: "{{ galaxy_config_dir }}/activation-email.html"
   - src: "{{ galaxy_config_file_src_dir }}/config/activation-email.txt"
     dest: "{{ galaxy_config_dir }}/activation-email.txt"
+  - src: "{{ galaxy_config_file_src_dir }}/config/local_tool_data_table_conf.xml"
+    dest: "{{ galaxy_config_dir}}/local_tool_data_table_conf.xml"
```
Comment (cat-bro) [inline host_vars/dev.usegalaxy.org.au.yml:106]:
```suggestion
    dest: "{{ galaxy_config_dir }}/local_tool_data_table_conf.xml"
```

File: host_vars/dev.usegalaxy.org.au.yml [hunk 2]
```diff
@@ -167,6 +169,7 @@ host_galaxy_config: # renamed from __galaxy_config
     interactivetools_map: "{{ gie_proxy_sessions_path }}"
     cleanup_job: never
     job_config_file: "{{ galaxy_config_dir }}/job_conf.yml"
+    tool_data_table_config_path: "{{ galaxy_config_dir}}/local_tool_data_table_conf.xml,{{ galaxy_mutable_config_dir }}/shed_tool_data_table_conf.xml,/cvmfs/data.galaxyproject.org/byhand/location/tool_data_table_conf.xml,/cvmfs/data.galaxyproject.org/managed/location/tool_data_table_conf.xml"
```
Comment (cat-bro) [inline host_vars/dev.usegalaxy.org.au.yml:172]:
```suggestion
    tool_data_table_config_path: "{{ galaxy_config_dir }}/local_tool_data_table_conf.xml,{{ galaxy_mutable_config_dir }}/shed_tool_data_table_conf.xml,/cvmfs/data.galaxyproject.org/byhand/location/tool_data_table_conf.xml,/cvmfs/data.galaxyproject.org/managed/location/tool_data_table_conf.xml"
```

**Review by cat-bro — APPROVED** [review]

**Pull request merged**